drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/pages/ActionInsertFactFieldsPageViewImpl.ui.xml

```diff
@@ -26,90 +27,99 @@
-          <b:Popover placement="TOP" trigger="HOVER" title="{i18n.UpdateFact}" content="{i18n.UpdateDescription}">
-            <b:Icon type="QUESTION" addStyleNames="help-block"></b:Icon>
-          </b:Popover>
-        </gwt:HorizontalPanel>
-      </gwt:VerticalPanel>
-      <gwt:VerticalPanel ui:field="fieldDefinition" visible="false">
-        <gwt:HorizontalPanel ui:field="columnHeaderContainer" styleName="{res.css.wizardDTableFieldContainerValid} form-group">
-          <gwt:InlineHTML>
-            <font color="red">*</font>
-          </gwt:InlineHTML>
-          <b:Label text="{i18n.ColumnHeaderDescription}" styleName="{res.css.wizardDTableFields} help-block"/>
-          <b:TextBox ui:field="txtColumnHeader" enabled="false" styleName="{res.css.wizardDTableFields}"/>
-          <b:HelpBlock ui:field="txtColumnHeaderHelp" addStyleNames="{res.css.wizardDTableFields}" text="{i18n.DecisionTableWizardIncompleteActions}" visible="false"/>
-        </gwt:HorizontalPanel>
-        <gwt:VerticalPanel ui:field="criteriaExtendedEntry">
-          <gwt:HorizontalPanel styleName="{res.css.wizardDTableFieldContainerValid}">
-            <b:Label text="{i18n.optionalValueList}" styleName="{res.css.wizardDTableFields}"/>
-            <b:TextBox ui:field="txtValueList" enabled="false" styleName="{res.css.wizardDTableFields}"/>
-            <b:Popover placement="TOP" trigger="HOVER" title="{i18n.ValueList}" content="{i18n.ValueListsExplanation}">
-              <b:Icon type="QUESTION" addStyleNames="help-block"></b:Icon>
-            </b:Popover>
-          </gwt:HorizontalPanel>
-          <gwt:HorizontalPanel ui:field="defaultValueContainer" styleName="{res.css.wizardDTableFieldContainerValid}">
-            <b:Label text="{i18n.DefaultValue}:" styleName="{res.css.wizardDTableFields}"/>
-            <gwt:SimplePanel ui:field="defaultValueWidgetContainer" styleName="{res.css.wizardDTableFields}"/>
-          </gwt:HorizontalPanel>
+
+      <b:Form type="HORIZONTAL">
+        <gwt:VerticalPanel ui:field="patternDefinition" visible="false" width="100%">
+          <b:FormGroup ui:field="bindingContainer">
+            <pf:FormLabelHelp for="txtBinding" addStyleNames="col-lg-4" text="{i18n.Binding}:" helpTitle="{i18n.BindingFact}" helpContent="{i18n.BindingDescription}"/>
+            <gwt:FlowPanel addStyleNames="col-lg-7">
+              <guvnor:BindingTextBox ui:field="txtBinding" enabled="false"/>
+              <b:HelpBlock ui:field="txtBindingHelp" text="{i18n.DecisionTableWizardDuplicateBindings}" visible="false"/>
+            </gwt:FlowPanel>
+          </b:FormGroup>
+          <b:FormGroup>
+            <pf:FormLabelHelp for="chkLogicalInsert" addStyleNames="col-lg-4" text="{i18n.LogicallyInsert}:" helpTitle="{i18n.LogicallyInsert}" helpContent="{i18n.LogicallyAssertAFactTheFactWillBeDeletedWhenTheSupportingEvidenceIsRemoved}"/>
+            <gwt:FlowPanel addStyleNames="col-lg-7">
```

Comment:
For cases like this, would be preferable to use <b:Column size="LG_7">. I still have doubts which size is actually the best for us MD or LG, but that is a different topic.